### PR TITLE
Created 4.5.0-rtm branch and updated releaseLabel

### PR DESCRIPTION
**Pull request by jainaashish**
Created 4.5.0-rtm branch and updated releaseLabel

@rrelyea

### CODE REVIEW

File: build/config.props [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <!-- We need to update this netcoreassembly build number with every milestone to workaround any breaking api
     changes we might have made-->
     <NetCoreAssemblyBuildNumber Condition=" '$(NetCoreAssemblyBuildNumber)' == '' ">2</NetCoreAssemblyBuildNumber>
```
Comment (emgarten) [inline build/config.props]: nit: should this go back to 0?

**Review by emgarten — APPROVED** [review]

File: build/config.props (hunk 1 above)
Comment (jainaashish) [reply build/config.props]: I think it should...

Comment (nkolev92) [reply build/config.props]: I don't think it matters too much, as long as it's a version that's different from the ones we have shipped. 

was it 0 at any point during the life of the 4.5 version?

Comment (emgarten) [reply build/config.props]: is this still used in the assembly version? I was assuming that it wasn't used at all. Now I'm more confused as to what it should be. Maybe it should go to 3 then?

Comment (nkolev92) [reply build/config.props]: Yeah it is still used. 
We basically have a different one each preview to work around the issues we have with the sdk/cli insertions. 

Not sure if we have ever used a 0, but afaik it should be fine as long as it's different from what we've had before. 

/cc 
@rrelyea

**Pull request merged**